pipenv/core.py

```diff
@@ -1128,6 +1128,12 @@ def do_lock(
     # Add default dependencies to lockfile.
     for dep in results:
         # Add version information to lockfile.
+        pipfile_version = project.packages[dep['name']] if dep['name'] in project.packages else None
```

Comment:
Does `project.packages.get(dep['name'])` work?

Reply:
Yes despite being a toml object this works fine